src/App.js

```diff
@@ -3,12 +3,14 @@ import './App.css';
 import AllArticles from './components/AllArticles';
 import Home from './components/Home';
 import ArticlesByTopic from './components/ArticlesByTopic';
+import ArticleById from './components/ArticleById';
 
 function App() {
   return (
     <BrowserRouter>
     <div className="App">
-      <nav>
+      <h1>NC-News</h1>
+      <nav className='section__articles'>
         <Link to='/'>Homepage </Link>
         <Link to='/articles'>View all articles</Link>
         <Link to='/articles/coding'>View coding articles</Link>
```

Comment:
These hardcoded links could be done by mapping data from a fetch to the topics in the API. What happens if you add or remove any topics?